openapi/access.yaml

```diff
@@ -541,14 +541,6 @@ components:
         _expandable:
           type: object
           properties:
-            proposal_key:
-              type: string
-            authorizers:
-              type: string
-            payload_signatures:
-              type: string
-            envelope_signatures:
-              type: string
```

Comment:
If we remove these as expandable fields, we need to add them as `required` above.

Reply:
Similar for the TransactionResult

Reply:
ahh! good catch.fixing it now

Reply:
done